src/Tools/dotnet-monitor/Strings.resx

```diff
@@ -267,6 +267,13 @@
 1. parameterValue: The value in the parameter that was rejected
 2. minBytes: The minimum number of bytes acceptable
 3. maxBytes: The maximum number of bytes acceptable</comment>
+  </data>
+  <data name="ErrorMessage_RegularExpressionDoesNotMatch" xml:space="preserve">
+    <value>The field {0} must match the regular expression '{1}'.</value>
```

Comment:
nit: While I think this is ok, it might actually be clearer to give a couple of examples. E.g.:
"The field {0} must be a status code or status code range between 100 and 500. E.g. 200, 400-500